src/extract_gpuinfo_tpu.c

```diff
@@ -0,0 +1,290 @@
+/*
+ *
+ * Copyright (C) 2022 Maxime Schmitt <maxime.schmitt91@gmail.com>
```

Comment:
I'm flattered but you can put your own copyright 😉

Reply:
Right :slightly_smiling_face: